src/frontend/mame/ui/miscmenu.h

```diff
@@ -15,10 +15,17 @@
 
 #include "crsshair.h"
 #include "emuopts.h"
+#include "modules/lib/osdobj_common.h"
 
 #include <utility>
 #include <vector>
 
+#ifdef WIN32
+#include "windows/winmain.h"
```

Comment:
Nope, anything that requires mame/ui to know anything about the OSD layout is completely unacceptable.  You'll need a cleaner way than this.